source/index.html

```diff
@@ -66,8 +71,8 @@
                         "
                                 />
                             </g>
-                            <text id="countdownText" x="26" y="50">25:00</text>
-                        </svg>
+                            <text id="countdownText" x="26" y="50" style="font-style: Roboto, sans-serif;">25:00</text>
```

Comment:
What's the reason for using the style attribute here over a separate CSS rule?

Reply:
I did a separate CSS rule because having an import URL in the css files makes another http request.  If you have multiple css files, then the css can't run in parallel. As a result, I downloaded the fonts, so we don't have to do a separate http request. However, svgs didn't like the downloaded fonts for some reason, and including the import url in the css won't make sense since I already downloaded it. Thus, I just included in the html instead. This is kind of a 'band aid' fix.

Reply:
Ahh makes sense, good work with the details